cmd/image/qcow2ova/prep/templates.go

```diff
@@ -157,11 +157,13 @@ write_files:
       set -e
       for i in /dev/sd[a-z]; do
         partprobe $i || true
-        growpart $i 2 || true
+        part=$(partprobe -s $i | awk '{print $NF}')
```

Comment:
again here also we are selecting the last partition to grow which is true in most of the cases but I'm wondering if we can write some logic so that it lists the linux partitions and only grow the last one, wdys @sudeeshjohn ?